src/components/Layout/styles.css

```
@@ -48,6 +48,36 @@
   grid-column: center-start / center-end;
 }
 
+/* HUGE */
```

Just "Huge" is a bit vague here IMO.
```suggestion
/* HUGE SCREENS */
```